src/uct/ib/configure.m4

```diff
@@ -353,12 +362,15 @@ AS_IF([test "x$with_ib" == xyes],
            ])
        AS_IF([test "x$with_ib_hw_tm" = xexp],
            [AC_DEFINE([IBV_EXP_HW_TM], 1, [IB Tag Matching support (EXP)])
```

Comment:
is IBV_EXP_HW_TM still needed? Looks like you check particular exp symbols when needed